src/aes.rs

```diff
@@ -248,6 +248,85 @@ pub fn mix_columns(input: &[u8; 16]) -> Option<[u8; 16]> {
     Some(ret)
 }
 
+fn gmix_column_c(input: &[UInt8Gadget; 4]) -> Option<[UInt8Gadget; 4]> {
+    let mut b: Vec<UInt8Gadget> = Vec::new();
+    /* The array 'a' is simply a copy of the input array 'r'
+     * The array 'b' is each element of the array 'a' multiplied by 2
+     * in Rijndael's Galois field
+     * a[n] ^ b[n] is element n multiplied by 3 in Rijndael's Galois field */
+
```

Comment:
Let's either remove or update these comments with their meaning in our function (we don't have an array `a`, and I'm assuming `r` is `input` in our case?)